### ISSUE TITLE
CORS error while trying to register

### ISSUE THREAD

**Issue by Elie-Soued** [gothinkster/angular-realworld-example-app#299]
**Steps to reproduce the issue:**
1- Clone repo
2-Run yarn install
3- run ng serve
4-go to the signup section
5-Fill the section with username, email and password
6- click the submit button

**Actual**
I am getting the below error:
Access to XMLHttpRequest at 'https://api.realworld.io/api/users' from origin 'http://localhost:4200' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Expected**
I can register.

**More notes:**
I have added the "Access-Control-Allow-Origin" header in the call but I am still getting the same error.

`  register(credentials: {
    username: string;
    email: string;
    password: string;
  }): Observable<{ user: User }> {
    
    return this.http
      .post<{ user: User }>("/users", { user: credentials },{headers : {"Access-Control-Allow-Origin" : "*"}})
      .pipe(tap(({ user }) => this.setAuth(user)));
  }`

**Comment by thien-nhat** [comment]
Yes, I also have this error, how to solve it. And it also shows: `[https://api.realworld.io/api/users'](https://api.realworld.io/api/users)` has the connection timeout

**Comment by datasmithadvtech** [comment]
I am encountering this error as well.
Where can i get the server code to run locally?

**Comment by ttyw-code** [comment]
Yes, I also have this error, how to solve it.

**Comment by geromegrignon** [comment]
The organization maintenance has been stopped, as explained here: https://github.com/gothinkster/realworld/issues/1611

**Comment by Thiagoow** [comment]
Geeh, that's sad 😕 but also very understandable. Wish you the best @geromegrignon!
Could please you share the repo of the API implemented on this specific project so we can run it locally?
